README.md

```diff
@@ -1,2 +1,53 @@
-# _template
-Template repo with README and CI settings
+# LV SJPG DECODER 
+- (first commit)
+
+# Overview:
+  - SJPG is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
```

Comment:
I'd add a line like "sjpg is a custom format based on the standard jpg" to be sure people will understand what it's about.

Reply:
Why was it marked as resolved?